### PR TITLE
Adding Error Telemetry for Rename and Delete Operations

### PR DESCRIPTION
**Pull request by ritikramuka**
In this PR I added telemetry log for rename and delete operations, in order to get number of users using these functionalities..

### CODE REVIEW

File: src/web/client/dal/fileSystemProvider.ts [hunk 1]
```diff
@@ -242,11 +244,29 @@ export class PortalsFS implements vscode.FileSystemProvider {
     }
 
     async rename(): Promise<void> {
-        throw new Error("Method not implemented.");
+        oneDSLoggerWrapper.getLogger().traceError(
+            telemetryEventNames.WEB_EXTENSION_RENAME_NOT_SUPPORTED,
+            WEB_EXTENSION_METHOD_NOT_IMPLEMENTED,
+            { name: telemetryEventNames.WEB_EXTENSION_RENAME_NOT_SUPPORTED, message: WEB_EXTENSION_METHOD_NOT_IMPLEMENTED} as Error,
```
Comment (tyaginidhi) [inline src/web/client/dal/fileSystemProvider.ts:250]: why is this relogging of eventName neded here?

File: src/web/client/dal/fileSystemProvider.ts [hunk 2]
```diff
@@ -242,11 +244,29 @@ export class PortalsFS implements vscode.FileSystemProvider {
...
+        );
+        throw new Error(WEB_EXTENSION_METHOD_NOT_IMPLEMENTED);
     }
 
     async delete(): Promise<void> {
-        throw new Error("Method not implemented.");
+        oneDSLoggerWrapper.getLogger().traceError(
+            telemetryEventNames.WEB_EXTENSION_DELETE_NOT_SUPPORTED,
+            WEB_EXTENSION_METHOD_NOT_IMPLEMENTED,
+            { name: telemetryEventNames.WEB_EXTENSION_DELETE_NOT_SUPPORTED, message: WEB_EXTENSION_METHOD_NOT_IMPLEMENTED} as Error
```
Comment (tyaginidhi) [inline src/web/client/dal/fileSystemProvider.ts:263]: same as above

File: src/web/client/dal/fileSystemProvider.ts (hunk 1 above)
Comment (ritikramuka) [reply src/web/client/dal/fileSystemProvider.ts]: as per conversation with @BidishaMS this telemetry logging in oneDSLogger is not required hence reverting the changes.

File: src/web/client/dal/fileSystemProvider.ts (hunk 2 above)
Comment (ritikramuka) [reply src/web/client/dal/fileSystemProvider.ts]: can you refer to the above for this too.

**Review by tyaginidhi — APPROVED** [review]

**Review by BidishaMS — APPROVED** [review]

**Pull request merged**